server/protocols/protocols-smtp/src/main/java/org/apache/james/smtpserver/futurerelease/FutureReleaseMailParameterHook.java

```diff
@@ -52,9 +61,9 @@ public HookResult doMailParameter(SMTPSession session, String paramName, String
         if (session.getUsername() == null) {
             LOGGER.debug("Needs to be logged in in order to use future release extension");
             return HookResult.builder()
-                .hookReturnCode(HookReturnCode.deny())
-                .smtpDescription("Needs to be logged in in order to use future release extension")
-                .build();
+                    .hookReturnCode(HookReturnCode.deny())
+                    .smtpDescription("Needs to be logged in in order to use future release extension")
+                    .build();
```

Comment:
IMO do not change the old indent (check your indent settings).

1 Tab (4 spaces) is good enough, not 2 Tab.